packages/data-sql/src/converter/index.test.ts

```diff
@@ -117,3 +117,97 @@ test('Get selects with a limit and offset', () => {
 
 	expect(res).toStrictEqual(expected);
 });
+
+test('Get selects with a sort', () => {
```

Comment:
I'd probably use a description like
```suggestion
test('Convert select query with sort modifier', () => {
```